go/cmd/vtctldclient/command/vreplication/mount/mount.go

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migrate
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	"vitess.io/vitess/go/cmd/vtctldclient/command/vreplication/common"
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+var (
+	// mount is the base command for all actions related to the mount action.
+	mount = &cobra.Command{
+		Use:                   "Mount [command] [command-flags]",
+		Short:                 "Mount is used to link external Vitess clusters to the current cluster to migrate data into.",
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"mount"},
+		Args:                  cobra.ExactArgs(1),
+	}
+)
+
+var MountOptions struct {
+	TopoType   string
+	TopoServer string
+	TopoRoot   string
+}
+
+func GetRegisterCommand() *cobra.Command {
```

Comment:
Curious why we're using this pattern here?

Reply:
Unexported. Changed to `var`: this felt better, but not how we are doing it elsewhere